pubspec.yaml

```diff
@@ -44,7 +46,7 @@ flutter:
   # To add assets to your application, add an assets section, like this:
   assets:
     - assets/logo_enei.png
-    - .env
+    - .env.sample
```

Comment:
This should be .env. The file .env.sample is an example how to build a .env file.

Reply:
ok, i will update this

Reply:
done